docs/source/AdministratorGuide/Resources/Catalog/index.rst

```diff
@@ -70,6 +65,25 @@ Then, each catalog should have a few (case-sensitive) options defined:
 * `AccessType`: `Read`/`Write`/`Read-Write`. No default, must be defined. This defines if the catalog is read-only, write only or both.
 * `Master`: see :ref:`masterCatalog`
 
+For example::
+
+
+   Catalogs
+   {
+
+      FileCatalog
+      {
+        AccessType = Read-Write
+        Status = Active
+        Master = True
+      }
+      BookkeepingDB
```

Comment:
Add a comment that this is not in DIRAC, but it's "yet another catalog", here as example, from an extension.